src/encrypt.cpp

```diff
@@ -1,12 +1,12 @@
 #include "pch.h"
 
 
-// [[Rcpp::export]]
-String encrypt_password(String password, String user) {
-  char* encrypted = PQencryptPassword(password.get_cstring(), user.get_cstring());
-
-  String copy(encrypted);
-  PQfreemem(encrypted);
-
-  return copy;
+[[cpp11::register]]
+cpp11::r_string encrypt_password(cpp11::r_string password, cpp11::r_string user) {
```

Comment:
Would the following work too:

```suggestion
std::string encrypt_password(std::string password, std::string user) {
```

Reply:
Yes, it will work. 
I have updated, to have more consistent code.